assets/scss/utilities/_spacing.scss

```diff
@@ -56,4 +58,20 @@
 
 .has-reset-padding {
   padding: 0;
+}
+
+.has-gutter-padding {
```

Comment:
Just a personal opinion, but when I think "gutter," I think about space between columns. This seems like it's more for adding space around an entire element as it relates to the edge of the window, not other elements. But feel free to override me. Regardless, this class will be very useful!